service/elastigroup/providers/azure/azure.go

```diff
@@ -135,8 +135,8 @@ type Dimension struct {
 }
 
 type Strategy struct {
-	LowPriorityPercentage *float64  `json:"lowPriorityPercentage,omitempty"`
-	OnDemandCount         *int      `json:"OnDemandCount,omitempty"`
+	LowPriorityPercentage *int      `json:"lowPriorityPercentage,omitempty"`
```

Comment:
Are these changes specifically to match the REST API?  Another reason?

Reply:
LowPriorityPercentage is an integer in Optimizer and the Spotinst API docs.